dali/core/exec/tasking/scheduler.cc

```diff
@@ -63,8 +63,10 @@ void Scheduler::Notify(Waitable *w) {
       // Otherwise, we have to re-check it.
       if (!is_completion_event && !w->IsAcquirable())
         break;
-      if (task->Ready())
+      if (task->Ready()) {
+        new_ready++;
```

Comment:
What does it change?

Reply:
Hopefully nothing... I even wonder if this condition can ever be true :upside_down_face: